pkg/kv/kvserver/store_create_replica.go

```diff
@@ -174,52 +211,19 @@ func (s *Store) tryGetOrCreateReplica(
 		return nil, false, &roachpb.RaftGroupDeletedError{}
 	}
 
-	// Create a new replica and lock it for raft processing.
+	// Create a new uninitialized replica and lock it for raft processing.
```

Comment:
the condition `replicaID != 0` is redundant, right? That's the replicaID we're creating the uninited replica for, which certainly should be nonzero. Looks like this dates back to preemptive snapshots and never got cleaned up?

Reply:
Right. Removed it.